data/pg312.json

```
@@ -0,0 +1,18 @@
+{
+  "name": "Padma",
+  "bio": "Tech enthusiast | Learner ",
+  "links": [
+    {
+      "group": "Personal profile",
+      "name": "Github profile",
+      "url": "https://github.com/pg312/",
+      "icon": "FaLink"
+    },
+    {
+      "group": "Personal profile",
+      "name": "Medium Reading Library",
+      "url": "https://medium.com/@bvpadmavati/list/java-2f1d37cc75b6",
+      "icon": "FaLink"
+    },
```

Please remove this trailing comma! 

```suggestion
    }
```